app-text/asciidoc/asciidoc-9.0.0_rc1.ebuild

```diff
@@ -0,0 +1,92 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} pypy3 )
+
+inherit autotools python-single-r1 readme.gentoo-r1
+
+DESCRIPTION="A plain text human readable/writable document format"
+HOMEPAGE="http://asciidoc.org/ https://github.com/asciidoc/asciidoc-py3/"
+SRC_URI="https://github.com/${PN}/${PN}-py3/archive/9.0.0rc1.tar.gz -> ${P}.tar.gz"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~riscv ~s390 ~sh ~sparc ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos ~sparc-solaris ~sparc64-solaris ~x64-solaris ~x86-solaris"
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="examples graphviz highlight test"
+RESTRICT="!test? ( test )"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+RDEPEND="
+	app-text/docbook-xml-dtd:4.5
+	>=app-text/docbook-xsl-stylesheets-1.75
+	dev-libs/libxslt
+	dev-libs/libxml2
+	${PYTHON_DEPS}
+	graphviz? ( media-gfx/graphviz )
+	highlight? (
+		|| (
+			dev-util/source-highlight
+			dev-python/pygments[${PYTHON_USEDEP}]
+			app-text/highlight
+		)
+	)"
+DEPEND="
+	test? (
+		app-text/dvipng
+		dev-texlive/texlive-latex
+		dev-util/source-highlight
+		media-gfx/graphviz
+		media-gfx/imagemagick
+		media-sound/lilypond
```

Comment:
This seems weird...

Reply:
What do you mean?  Are you referring to the difference in dependencies between USE="test" and otherwise?  If so: While asciidoc supports three alternatives for source highlighting, the reference output in the tests were created with source-highlight, so the tests specifically require that.

Reply:
`media-sound/lilypond` but I didnt really dig into upstream what does it do.

Reply:
Oh, I wasn't sure myself (I've heard of lilypond, but never used it myself), see the file at /usr/share/doc/asciidoc-8.6.10_p20181016/examples/music-filter.txt.  Looks like another candidate for optfeature.